bodyparsers/msgpack/src/component.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/bodyparser-msgpack
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Binding, Component} from '@loopback/core';
+import {createBodyParserBinding} from '@loopback/rest';
+import {
+  DefaultMsgPackParserLibrary,
+  MsgPackBodyParser,
+  MsgPackBodyParserBindings,
+} from '.';
+
+// FIXME(achrinza): Workaround for https://github.com/microsoft/rushstack/pull/1867
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import * as loopbackCore from '@loopback/core';
+
+export class MsgPackBodyParserComponent implements Component {
+  bindings = [
```

Comment:
Let's explicitly mark `bindings: Binding[]=` to work around the apidocs generation limitation - which fails `verify-docs` CI task.